static/js/chart/draw.ts

```diff
@@ -469,37 +469,49 @@ function drawLineChart(
   }
 }
 
-interface PlotParams {
-  // Label to color.
-  colors: { [key: string]: string };
-  // Label to dash style.
-  dashes: { [key: string]: string };
-  title: { [key: string]: string };
+interface Style {
+  color: string;
+  dash: string;
 }
 
 /**
- * Return color and dash style given place names and stats names.
+ * Return color and dash style given place names and stats var dcids.
```

Comment:
could you clarify what the key is for the return value. or perhaps the key should also be part of the interface?

Reply:
it's also unclear from just reading this function (and comment) how to use the returned dictionary. if we keep the interface as is, please clarify.

Reply:
refactored based on the one pager

Reply:
thanks! can you link that here? or add a gist of it to the comment above.

some things to point out are the keys in the returned value. it will fail if stat var display titles diverge from the actual stat var. or maybe clarify that display names are required?

Reply:
Yea, this is not ideal, added comments to clarify. The issue is due to sharing the color information with the stats var chips component. Ideally, if we can draw the chips as svg inside the drawGroupLineChart() function, this will be cleaner.

Reply:
Yea, gotcha.